.github/workflows/integration-tests.yml

```diff
@@ -53,8 +53,9 @@ jobs:
       - name: Install ccm
         # The tool used by unit tests to setup database https://github.com/riptano/ccm
         run: |
-          wget https://bootstrap.pypa.io/ez_setup.py -O - | python
-          pip install ccm
+          git clone https://github.com/riptano/ccm.git --depth 1
+          cd ccm
+          sudo ./setup.py install
```

Comment:
I can see that you went through "debugging on CI"...
That must have been an enriching, yet tough experience. :grin:

Reply:
Well...